Source/RapyutaSimulationPlugins/Private/Core/RRNetworkPlayerController.cpp

```diff
@@ -88,9 +88,10 @@ void ARRNetworkPlayerController::InitSimStateClientROS2()
     // Init SimStateClient's [ROS2Node] & [ClockPublisher]
     UWorld* currentWorld = GetWorld();
     SimStateClientROS2Node = currentWorld->SpawnActor<AROS2Node>();
+    SimStateClientROS2Node->AttachToActor(this, FAttachmentTransformRules::KeepRelativeTransform);
```

Comment:
That might be a good idea to make it a component since we always seem to have an actor that defines all the services/msgs and then its initialized in the ROS2Node, could be easier if the ROS2node was a component and just picked it up from the actor/object it was attached it